### PR TITLE
doc(test): Add the test command line in readme and how to use it

### PR DESCRIPTION
**Pull request by vladimirkosmala**
## 📑 Description
I needed to read the code to understand how to run the tests.
This commit will directly show in the README how to set the Bunny's credentials.

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -96,6 +96,16 @@ Don't forget to give the project a star! Thanks again!
 4. Push to the Branch (`git push origin feature/AmazingFeature`)
 5. Open a Pull Request
 
+## Testing
+
+The project is tested with vitest and requires you to have a test library on your BunnyCDN account to send commands to the Bunny API. Warning: all videos will be deleted during testing.
```
Comment (dan-online) [inline README.md:101]:
```suggestion
The project is tested with `vitest` and requires you to have a library on your BunnyCDN account to send commands to the Bunny API. 

> Warning: all videos will be deleted during testing.

### Keys
```

File: README.md [hunk 2]
```diff
@@ -96,6 +96,16 @@ Don't forget to give the project a star! Thanks again!
...
+
+```sh
+BUNNY_VIDEO_LIBRARY=xxx BUNNY_API_KEY=xxx npm test
+```
+
+Also, IGNORE_PRUNE=1 can be used to keep the video in Bunny at the end of the tests for further manual checking.
```
Comment (dan-online) [inline README.md:107]:
```suggestion
Also, `IGNORE_PRUNE=1` can be used to keep the testing video at the end of the tests for further manual checking.
```

File: README.md (hunk 1 above)
Comment (dan-online) [inline README.md:101]:
```suggestion
The project is tested with `vitest` and requires you to have a library on your BunnyCDN account to send commands to the Bunny API. 

> Warning: all videos will be deleted during testing.

### Keys
```

**Review by dan-online — CHANGES_REQUESTED** [review]
Some phrasing and formatting changes needed

**Comment by vladimirkosmala** [comment]
> Some phrasing and formatting changes needed

This looks fine

**Review by dan-online — APPROVED** [review]

**Pull request merged**